### ISSUE TITLE
Xcode Error "nil class SpringBoard"

### ISSUE THREAD

**Issue by Raizelian** [iam4x/pokemongo-webspoof#49]
The build on Xcode is successfull but when I try to run the app on my phone the screen remains white and Xcode reports a "nil class Springboard" error. I tried googling what it meant and it has something to do with "unhooking from the springboard while debugging"? I'm confused.

![nil class springboard](https://cloud.githubusercontent.com/assets/18742036/16951612/032dc6ee-4dc6-11e6-94ff-fd405e4b34fc.png)

Also, I'm on iOS 8.4 and had to change the related option in the Xcode project to make it run, but I doubt the error could come from that... Maybe it's because my phone is jailbroken?

Any ideas?

**Comment by iam4x** [comment]
I have no experience with iOS8, ping @othierry

**Comment by AbelIngrand** [comment]
Yes the error comes from your Jailbreak, you have a Tweak installed that tampers with the Springboard. Try uninstalling tweaks and trying again.

**Comment by Raizelian** [comment]
Is there some way to single out which tweak is the one causing problems? I have lots of them installed, and would hate to find out by trial and error.

**Comment by AbelIngrand** [comment]
Not with the Xcode log. But it's certainly a tweak that modifies the home screen or the control center or Notification Center.

The warning is meaningless, you can ignore it.

**Comment by Raizelian** [comment]
So I solved it. Apparently **the "nil class" error can be ignored**.
I then misunderstood what it meant to "Check the Auto update Xcode location in the app" because I thought something had to appear on my iPhone screen, but the screen remained white; it is on the _Mac_ app, instead, that the option should be switched on. My mistake there.

Also, I had two "autoclick Xcode" errors once I managed to make everything run, and one of them was fixed by granting privileges to the app in the Accessibility preferences.

The other one, this:
![screen shot 2016-07-21 at 00 14 00](https://cloud.githubusercontent.com/assets/18742036/17005279/a72c8b86-4ed9-11e6-8703-81bf47adba15.png)
seems to be harmless, as everything works as it's supposed to. The connection is awful, but I think that's just the servers - as usual.

Thanks to everyone.

P.s.: Little suggestion: maybe there could be a way to make the game avatar move diagonally if two arrows are pressed at once? Like the down and right arrow to go diagonally in an east direction, for example. In the meanwhile, great job on this.
